<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>PLQE</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Laser start</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="laser_start_spinBox">
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="value">
         <double>200.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Laser stop</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="laser_stop_spinBox">
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="value">
         <double>400.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Emission start</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="emission_start_spinBox">
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="value">
         <double>700.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Emission stop</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="emission_stop_spinBox">
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="value">
         <double>800.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>PLQE Percent</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="plqe_label">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QPushButton" name="calculate_plqe_pushButton">
        <property name="text">
         <string>Calculate PLQE</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="clear_pushButton">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="plot_pushButton">
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="load_data_pushButton">
     <property name="text">
      <string>Load data</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="PlotWidget" name="plotWidget"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
